1-quality.rst

```diff
@@ -148,6 +149,17 @@ The paired sequences output by this set of commands will be in the
 files ending in ``.qc.fq.gz``, with any orphaned sequences all together
 in ``orphans.qc.fq.gz``.
 
+Save these trimmed reads for later:
+
+::
+
+      mkdir /mnt/work/data/trim/
```

Comment:
should we create directories for the output of other steps here? I think I'd prefer not, because it can make it unclear what step is doing what and where the data files for each step come from; so maybe a new principle, "each step operates only within one working directory"?